configman/def_sources/for_argparse.py

```diff
@@ -573,7 +575,7 @@ def parse_args(self, args=None, namespace=None):
             values_source_list=self.value_source_list,
             argv_source=args,
             app_name=self.prog,
-            app_version=self.version,
+#            app_version=self.version,
```

Comment:
Just uncommented and re-ran.
I got this:

```
======================================================================
ERROR: test_parser_setup (configman.tests.test_def_for_argparse.TestCaseForDefSourceArgparse)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/asasaki/src/releng/configman/configman/tests/test_def_for_argparse.py", line 250, in test_parser_setup
    config = parser.parse_args(args=[])
  File "/Users/asasaki/src/releng/configman/configman/def_sources/for_argparse.py", line 578, in parse_args
    app_version=self.version,
AttributeError: 'ArgumentParser' object has no attribute 'version'
```

Want me to add it back in `if six.PY2` ?